homeassistant/components/elkm1/climate.py

```diff
@@ -198,7 +204,7 @@ async def async_set_temperature(self, **kwargs):
         if high_temp is not None:
             self._element.set(ThermostatSetting.COOL_SETPOINT.value, round(high_temp))
 
-    def _element_changed(self, element, changeset):
-        self._state = ELK_TO_HASS_HVAC_MODES.get(self._element.mode)
+    def _element_changed(self, element: Element, changeset: Any) -> None:
+        self._state = ELK_TO_HASS_HVAC_MODES.get(self._element.mode, HVAC_MODE_OFF)
```

Comment:
```suggestion
        self._state = ELK_TO_HASS_HVAC_MODES[self._element.mode]
```

Per your comment in the opening text, it looks like it should be this